packages/mux-player/src/index.ts

```diff
@@ -271,22 +271,32 @@ class MuxPlayerElement extends VideoApiElement {
     // from video.onerror. This allows us to simulate errors from the outside.
     this.addEventListener('error', onError);
 
-    this.video?.addEventListener('error', (event: Event) => {
-      let { detail: error }: { detail: any } = event as CustomEvent;
+    if (this.video) {
+      this.video.errorTranslator = (errorEvent: ErrorEvent = {}) => {
+        if (!this.video?.error) return errorEvent;
 
-      // If it is a hls.js error event there will be an error object in the event.
-      // If it is a native video error event there will be no error object.
-      if (!error) {
-        const { message, code } = this.video?.error ?? {};
-        error = new MediaError(message, code);
-      }
+        const { devlog } = getErrorLogs(
+          this.video?.error,
```

Comment:
last error can be accessed here on `this.video.error` similar to 
https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/error